README.md

```diff
@@ -108,3 +108,28 @@ for more information about submitting pull requests or changes to EarthPy.
 
 [BSD-3](https://github.com/earthlab/earthpy/blob/master/LICENSE)
 
+<<<<<<< HEAD
```

Comment:
@nkorinek this looks like you had a merge conflict that was not cleaned up properly. the only change to the readme should be adding a URL. nothing else should be different. please undo this change as well.